src/custom/utils/operator/errors/MetadataError.ts

```diff
@@ -1,23 +1,27 @@
 type ApiActionType = 'update'
 
-export interface ApiErrorObject {
-  errorType: ApiErrorCodes
+export interface MetadataApiErrorObject {
+  errorType: MetadataApiErrorCodes
   description: string
 }
 
 // TODO: Validate/Update error code when the API is updated
-export enum ApiErrorCodes {
+export enum MetadataApiErrorCodes {
+  AddressNotFound = 'AddressNotFound',
+  UNHANDLED_GET_ERROR = 'UNHANDLED_GET_ERROR',
   UNHANDLED_UPLOAD_ERROR = 'UNHANDLED_UPLOAD_ERROR',
 }
```

Comment:
Is there any reason why we are using different cases for enum variables?
How about `ADDRESS_NOT_FOUND`, unless it makes anything wrong?

Reply:
I followed the existing conventions. Take a look at `OperatorError` and `QuoteError` files